webapps-backup/src/main/java/io/camunda/webapps/backup/repository/elasticsearch/ElasticsearchBackupRepository.java

```diff
@@ -312,10 +314,9 @@ public List<SnapshotInfo> findSnapshots(final String repositoryName, final Long
               "Encountered an error connecting to Elasticsearch while searching for snapshots. Repository name: [%s].",
               repositoryName);
       throw new BackupRepositoryConnectionException(reason, ex);
-    } catch (final Exception e) {
+    } catch (final ElasticsearchException e) {
```

Comment:
Previously the exceptions reaised from the try block where swallowed by this Exception clause and not rethrown